slack_today_i_did/parser.py

```diff
@@ -72,6 +72,10 @@ def eval(tokens, known_functions, default_args=None):
         first_function_name = 'error-help'
         args.append(('NO_TOKENS', str))
     # when we have stuff to work with!
+    elif 'help' == tokens[0][1]:
```

Comment:
Trying to avoid hardcoding things in here as much as possible. I think the right approach long-term is to introduce function pointers, instead of the current eager evaluation, and provide more power to the commands instead